build/docker-images/HealthChecks.UI.Image/Extensions/IEndpointRouteBuilderExtensions.cs

```diff
@@ -0,0 +1,24 @@
+﻿using HealthChecks.UI.Image.Configuration;
+using HealthChecks.UI.Image.Extensions;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Routing;
+using Microsoft.Extensions.Configuration;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Microsoft.AspNetCore.Builder
+{
+    public static  class IEndpointRouteBuilderExtensions
+    {
+        public static IEndpointConventionBuilder MapDockerHealthchecksUI(this IEndpointRouteBuilder builder, IConfiguration configuration)
```

Comment:
```suggestion
        public static IEndpointConventionBuilder MapHealthchecksUI(this IEndpointRouteBuilder builder, IConfiguration configuration)
```

Reply:
In fact there is nothing special about Docker, just "standard" configuring.

Reply:
That method already exists. The intention is create a method to configure docker related stuff that can not be configured as the image is already biult

Reply:
> That method already exists in UI assembly with Microsoft.AspNetCore.Builder namespace

With different signature. It will be just another overload.

> to configure docker related stuff 

This is clear. But the code itself has nothing to do with Docker. This is the usual `IConfiguration`-based configuration. Such a name can confuse someone.

Reply:
Ok @sungam3r, it makes sense. updated